capitanodoc.coffee

```diff
@@ -0,0 +1,107 @@
+# coffeelint: disable=max_line_length
+
+module.exports =
+	title: 'Resin CLI Documentation'
+	introduction: '''
+		This tool allows you to interact with the resin.io api from the comfort of your command line.
+
+		Please make sure your system meets the requirements as specified in the [README](https://github.com/resin-io/resin-cli).
+
+		To get started download the CLI from npm.
+
+			$ npm install resin-cli -g
+
+		Then authenticate yourself:
+
+			$ resin login
+
+		Now you have access to all the commands referenced below.
+
+		## Proxy support
+
+		The CLI has the first-class support for HTTP(S) proxies.
```

Comment:
Extra 'the' here, and I'm not sure what 'first-class' means in terms of proxy support? I'd just drop it to "The CLI supports HTTP(S) proxies".